mmv1/templates/terraform/post_delete/private_cloud.go.erb

```diff
@@ -26,9 +26,19 @@ privateCloudPollRead := func(d *schema.ResourceData, meta interface{}) transport
                 RawURL: url,
                 UserAgent: userAgent,
             })
+            // if resource does not exist
             if err != nil {
+                log.Printf("[DEBUG] The Private cloud has been successfully deleted.")
```

Comment:
this if statement actually indicates there was an error during deletion.

Reply:
Noted, thanks